docs/index.html

```diff
@@ -76,6 +76,7 @@ <h3 class="mt-4">Some code examples:</h3>
       <li><a href="./seek-thumbs.html">Seek Thumbnails</a></li>
       <li><a href="./share-plugin.html">Share &amp; Download</a></li>
       <li><a href="./shoppable.html">Shoppable Videos</a></li>
+      <li><a href="./source-switcher.html">Source switcher</a></li>
```

Comment:
Should we also add an ESM example page? I'm ok with not having everything doubled, just that this is what we did with other examples

Reply:
we dont have yet E2E test for this page (just mocked for now) but I believe ESM should be added only if there is a reason for that (otherwise we are doubling everything), for now I will keep just one page but we need to decide about some rules here (will schedule meeting for us after gathering)